packages/cfd/src/Components/cfd-account-card.tsx

```diff
@@ -159,7 +159,7 @@ const CFDAccountCard = ({
     commission_message,
     descriptor,
     is_hovered,
-    existing_data,
+    existing_datas,
```

Comment:
Best to keep this variable name as it is, data is already plural 😄

Reply:
let's find a new name :) 
how about existing_data_arr? or anything that you think may describe the variable better.

Reply:
Will do.